redfish/thermal.go

```diff
@@ -24,82 +24,95 @@ const (
-	HotPluggable bool
-	// IndicatorLED shall contain the state of the indicator light associated with this fan.
-	IndicatorLED common.IndicatorLED
-	// Location shall contain the location information of the associated fan.
-	Location common.Location
-	// LowerThresholdCritical shall contain the value at which the Reading property is below the normal range but is
-	// not yet fatal. The value of the property shall use the same units as the Reading property.
-	LowerThresholdCritical int
-	// LowerThresholdFatal shall contain the value at which the Reading property is below the normal range and is
-	// fatal. The value of the property shall use the same units as the Reading property.
-	LowerThresholdFatal int
-	// LowerThresholdNonCritical shall contain the value at which the Reading property is below normal range. The value
-	// of the property shall use the same units as the Reading property.
-	LowerThresholdNonCritical int
-	// Manufacturer shall contain the name of the organization responsible for producing the fan. This organization may
-	// be the entity from whom the fan is purchased, but this is not necessarily true.
-	Manufacturer string
-	// MaxReadingRange shall indicate the highest possible value for the Reading property. The value of the property
-	// shall use the same units as the Reading property.
-	MaxReadingRange int
-	// MemberId shall contain the unique identifier for this member within an array. For services supporting Redfish
-	// v1.6 or higher, this value shall contain the zero-based array index.
-	MemberID string
-	// MinReadingRange shall indicate the lowest possible value for the Reading property. The value of the property
-	// shall use the same units as the Reading property.
-	MinReadingRange int
-	// Model shall contain the model information as defined by the manufacturer for the associated fan.
-	Model string
-	// Oem shall contain the OEM extensions. All values for properties that this object contains shall conform to the
-	// Redfish Specification-described requirements.
-	OEM json.RawMessage `json:"Oem"`
-	// PartNumber shall contain the part number as defined by the manufacturer for the associated fan.
-	PartNumber string
-	// PhysicalContext shall contain a description of the affected device or region within the chassis with which this
-	// fan is associated.
-	PhysicalContext PhysicalContext
-	// Reading shall contain the fan sensor reading.
-	Reading int
-	// ReadingUnits shall contain the units in which the fan reading and thresholds are measured.
-	ReadingUnits ReadingUnits
-	// Redundancy shall contain an array of links to the redundancy groups to which this fan belongs.
-	redundancy []string
-	// RedundancyCount is the number of Redundancy items.
-	RedundancyCount int `json:"Redundancy@odata.count"`
-	// RelatedItem shall contain an array of links to resources or objects that this fan services.
+
+	// Assembly contains a link to a resource of type Assembly
+	Assembly string `json:"Assembly"`
```

Comment:
This shouldn't be made public. Links like this should stay private, with `Assembly()` used to access the Assembly object.